crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -60,7 +60,9 @@ pub async fn publish_dht_ops_workflow(
 
     // Commit to the network
     for (basis, ops) in to_publish {
-        network.publish(true, basis, ops, None).await?;
+        if let Err(e) = network.publish(true, basis, ops, None).await {
+            tracing::info!(failed_to_send_publish = ?e);
```

Comment:
wouldn't this be at least a warning?

Reply:
No because it's pretty normal for a peer to be offline. It's not really an error and would just spam our warnings.